force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -39,18 +39,12 @@ public with sharing class ClassesObjectsMethods {
      * @return A list containing the length of the string, a substring of the string, and the string in lower case.
      */
     public static List<String> practiceStringMethods() {
-        // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
-        // Get a substring of the String of the first 5 characters
-        // Convert the String to lower case
-        
-        // Return the list with the string data
-        // In the first position add the string length
-        // In the second position add the substring
-        // In the third position add the string in lower case
-
-        return null; // Replace null with the variable you used to store the result
+        String brownNose = 'Hello, SalesForce!';
+        String length = String.valueOf(brownNose.length());
```

Comment:
Although this is very funny and I appreciate humor but  when naming variables, try to give them a name that explains what they are doing.  Nothing wrong with some cool funny names but think about if you were to come back to your code a year from now. Would someone know what the variable is doing right off the bat?